### ISSUE TITLE
Coud not  build js with flow types

### ISSUE THREAD

**Issue by retyui** [developit/microbundle#379]
```js
(flow-remove-types plugin) SyntaxError: Unexpected token (85:44)

at undefined:85:44
SyntaxError: Unexpected token (85:44)
    at Parser.pp$5.raise (/home/i/all-work/react-quick-pinch-zoom/node_modules/babylon/lib/index.js:4454:13)
    at Parser.pp.unexpected (/home/i/all-work/react-quick-pinch-zoom/node_modules/babylon/lib/index.js:1761:8)
    at Parser.pp$3.parseParenAndDistinguishExpression (/home/i/all-work/react-quick-pinch-zoom/node_modules/babylon/lib/index.js:3859:10)
    at Parser.pp$3.parseExprAtom (/home/i/all-work/react-quick-pinch-zoom/node_modules/babylon/lib/index.js:3709:19)
    at Parser.parseExprAtom (/home/i/all-work/react-quick-pinch-zoom/node_modules/babylon/lib/index.js:7238:22)
    at Parser.pp$3.parseExprSubscripts (/home/i/all-work/react-quick-pinch-zoom/node_modules/babylon/lib/index.js:3494:19)
    at Parser.pp$3.parseMaybeUnary (/home/i/all-work/react-quick-pinch-zoom/node_modules/babylon/lib/index.js:3474:19)
    at Parser.pp$3.parseExprOp (/home/i/all-work/react-quick-pinch-zoom/node_modules/babylon/lib/index.js:3435:42)
    at Parser.pp$3.parseExprOp (/home/i/all-work/react-quick-pinch-zoom/node_modules/babylon/lib/index.js:3438:19)
    at Parser.pp$3.parseExprOps (/home/i/all-work/react-quick-pinch-zoom/node_modules/babylon/lib/index.js:3408:17)

```

**Comment by retyui** [comment]
https://github.com/flowtype/flow-remove-types/issues/88

**Comment by retyui** [comment]
How about using [babel plugin](https://babeljs.io/docs/en/babel-plugin-transform-flow-strip-types) to remove flow types?

**Comment by developit** [comment]
That's a great idea! It would avoid an entire AST transform that we currently run on all .js files regardless of whether they have flow types!

**Comment by developit** [comment]
Fixed as of #362.
